src/utils.js

```diff
@@ -29,6 +31,9 @@ function _getBaseUrl(u, currentBaseUrl = '') {
 module.exports._getBaseUrl = _getBaseUrl;
 
 function _normalizeUrl(src, baseUrl) {
+  if (/^[/][/]/.test(src)) {
+    src = new URL(baseUrl || 'https://www.example.com').protocol + src;
```

Comment:
I don't think `https:` should be assumed in the missing `baseUrl` case.

If this is an error, maybe it should throw?